lib/runner.js

```diff
@@ -88,6 +89,8 @@ class Runner extends Emittery {
 			}
 
 			metadata.taskIndex = this.nextTaskIndex++;
+			const testInvocation = stackUtils.parseLine(stackUtils.captureString().split('\n')[0]);
```

Comment:
I'm not sure we should be capturing this at runtime. I'm a little concerned about any additional performance overhead, especially in files with lots of tests.